Remove client-cert references, refinements from 117 (#27)
egorbaty/draft-httpbis-secondary-server-certs: draft-egorbaty-httpbis-secondary-server-certs.md

## Invalid Certificates
Unacceptable certificates (expired, revoked, or insufficient to satisfy the
request) are not treated as stream or connection errors. This is typically not
an indication of a protocol failure. Clients SHOULD establish a new connection

> in an attempt to reach an authoritative server, in the case that they deem a

Reviewer: Minor nit
```suggestion
in an attempt to reach an authoritative server if they deem a
```
